src/components/Typography/BodyX/BodyX.module.css

```diff
@@ -33,3 +33,15 @@
   font-size: var(--typography-bodyL-fontSize, 18px);
   line-height: var(--typography-bodyL-lineHeight, 24px);
 }
+
+.bold {
+  color: var(--palette-text-neutral-bold, #464646);
+}
+
+/* This class is empty because the main color is managed via Ant Theme */
+.main {
```

Comment:
if is not needed, I suggest to remove to avoid breaking everywhere a "bodyX" is used

Reply:
From a semantic standpoint it makes sense to have the hierarchy value imho 

I agree on removing this class since it affects too many snapshots

Reply:
we can consider main as default and we don't need to distinguish by other type